c10/util/SymIntTable.cpp

```diff
@@ -0,0 +1,36 @@
+#include <ATen/core/SymbolicIntNode.h>
+
+
+namespace c10 {
+
+    size_t SymIntTable::addNode(std::shared_ptr<SymbolicIntNode> sin) {
```

Comment:
nit:
We use `int64_t` and `uint64_t` for `data_`.  Is it necessary to let `size_t` propagate as well?

Reply:
yeah I need to do a better job here to reduce implicit conversions